.vortex/docs/.utils/VideoRecorder.php

```
@@ -232,165 +228,50 @@ public function recordSession(string $cwd, string $cast_path, string $command, s
       'asciinema', 'rec',
       "--window-size=$size",
-      '--output-format=asciicast-v2',
       "--title=$title",
       "--command=$command",
       '--overwrite',
       $cast_path,
     ], $cwd, $env);
 
     if (!is_file($cast_path)) {
-      throw new RuntimeException("Recording produced no cast file at $cast_path");
+      throw new \RuntimeException("Recording produced no cast file at $cast_path");
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '244', column '17'). (undefined)

(MissingImport)